internal/provider/nordvpn/updater/models.go

```diff
@@ -83,6 +85,18 @@ func (s *serverData) region() (region string) {
 	return ""
 }
 
+// categories returns the list of categories for the server.
+func (s *serverData) categories() (categories []string) {
+	cats := make([]string, 0, len(s.Groups))
+
+	for _, group := range s.Groups {
+		if group.Type.Identifier == "legacy_group_category" {
+			cats = append(cats, group.Identifier)
+		}
```

Comment:
Quick question: why do we only take legacy group categories? Is there a new group category we can use instead?

Reply:
Ok! Should we use the `title` instead since it's a bit more user friendly?

Reply:
Using the title, so now it's one of: Dedicated IP, Double VPN, Onion Over VPN, P2P, Standard VPN servers